src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -255,7 +258,9 @@ String getLinker() {
         if (!useJavaFX) return linkFlags;
         List<String> answer = new ArrayList<>();
         answer.addAll(linkFlags);
-        answer.addAll(javafxLinkFlags);
+        if (projectConfiguration.getClasspath().contains("javafx-web")) {
+            javafxLinkFlags.add(javafxLinkFlags.indexOf(WL_NO_WHOLE_ARCHIVE) - 1, javafxWebLib);
```

Comment:
This is dangerous. It's better to fit the lib between whole-archive/no-whole-archive options, so that it keeps working in case there is more than 1 occurrence of whole-archive

Reply:
I've fixed this by adding it to the end.